tomcat/src/test/java/org/apache/coyote/http11/request/Http11RequestTest.java

```diff
@@ -0,0 +1,51 @@
+package org.apache.coyote.http11.request;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Http11RequestTest {
+
+    @Test
+    @DisplayName("요청에 따라 정적 파일 요청인지 아닌지 알 수 있다.")
+    void isStaticRequest_static() {
+        String staticRequest = """
+                GET /index.html HTTP/1.1
+                Host: localhost:8080
+                Connection: keep-alive
+                
+                
+                """;
+        InputStream inputStream = new ByteArrayInputStream(staticRequest.getBytes());
+        Http11Request request;
+        try {
+            request = Http11Request.from(inputStream);
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+        assertThat(request.isStaticRequest()).isTrue();
+    }
+
+    @Test
+    @DisplayName("요청에 따라 정적 파일 요청인지 아닌지 알 수 있다.")
+    void isStaticRequest_nonStatic() {
```

Comment:
```suggestion
    void isStaticRequest_nonStatic() throws IOException {
```
위와 같은 방법으로 try-catch를 사용하지 않고 테스트를 작성할 수 있을 것 같아요:)

Reply:
반영했습니다!!